config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
+  # get 'users/index'
+  # get 'users/show'
+  # get 'users/posts'
+  # get 'posts/index'
+  # get 'posts/show'
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you remove these commented routes.

Reply:
### Hi @julie-ify 
Thank you very much for reviewing my project.